test/e2e/ginkgo/multiple_cluster_deployment_test.go

```diff
@@ -78,6 +78,25 @@ var _ = ginkgo.Describe("end to end testing", func() {
 			util.ValidateNumDeployments(ctx, "wec2", wec2, ns, 1, testLabelAbsent)
 		})
 
+		
+		ginkgo.AfterEach(func(ctx context.Context) {
```

Comment:
We do cleanup before the next test, so that a failing test leaves state that can be examined. See https://github.com/shivansh-gohem/kubestellar/blob/0e69477f5c61e6f4da51cdea50d22d9442dfc16f/test/e2e/ginkgo/multiple_cluster_deployment_test.go#L50